### ISSUE TITLE
Error fast5Index.tmp and parse error reading ".bed.tmp"

### ISSUE THREAD

**Issue by acarmas1** [weir12/DeepEdit#10]
Hello,

I've been trying to use DeepEdit to detect inosines in my direct RNA reads. I've submitted this script to my server: 

bash /scratch/acarmas/AtoI/test_data/DeepEdit/Getting_Started/1.Reads_extract.sh \
--SiteFile /scratch/acarmas/AtoI/preprocessing/target_site_apis.bed \
--BamPath /scratch/acarmas/AtoI/preprocessing/head_rep1_assembly.sam \
--Fast5Dir /scratch/acarmas/AtoI/fast5/fastq_head_rep1/multi_to_single/Join \
--Output /scratch/acarmas/AtoI/preprocessing/head_rep1_assembly_target_site_reads.txt

However, I'm getting this error: 

grep: .fast5Index.tmp: Input/output error
grep: .fast5Index.tmp: Cannot send after transport endpoint shutdown
grep: .fast5Index.tmp: Cannot send after transport endpoint shutdown
grep: .fast5Index.tmp: Cannot send after transport endpoint shutdown
[bed_read] Parse error reading ".bed.tmp" at line 2
samtools view: Could not read file ".bed.tmp"
[bed_read] Parse error reading ".bed.tmp" at line 2

I was wondering if you could help me to troubleshoot it please.

Thank you. I'm looking forward to hearing from you.

Best,
Camila

**Comment by LongxianChen** [comment]
Because the script produces some .tmp files, so please ensure that you have write and read permission of your working directory in which you run this command.

**Comment by LongxianChen** [comment]
By the way, we updated the 2.feature_extract.py file. We recommend to use this newer version script.

**Comment by acarmas1** [comment]
Hello, thanks for your help, I was to able to read the bed.tmp. But I'm still getting the fast5index.tmp issues. I'm reindexing my fast5 with tombo. 

I do have a question regarding the 2.feature_extract.py. For tombo I'm using this parameters: --corrected-group RawGenomeCorrected_000 --basecall-group Basecall_1D_000. Since this is the basecall group of my reads. But in your documentation and test dataset is 001. What part of the 2.feature_extract.py script should be change to use it for 000.

**Comment by acarmas1** [comment]
Also I have another question. I regenerated the tombo resquiggle index and I've been running 1.read_extract script for 1 day. However, I just got this message. 
![image](https://github.com/weir12/DeepEdit/assets/33578163/6a0985e3-f2e2-490f-9a7b-53b32984f10c)
And this is how I submitted the job. 
![image](https://github.com/weir12/DeepEdit/assets/33578163/d66a1269-bfde-48dd-ad37-1c8fe3de32e7)

**Comment by acarmas1** [comment]
Hello, 
I'm still getting this error: 
![image](https://github.com/weir12/DeepEdit/assets/33578163/026a8b0f-524c-4f76-9c6e-2767dfeb94ef)
I was wondering if someone could help me to troubleshoot it, please.

**Comment by LongxianChen** [comment]
> Hello, I'm still getting this error: ![image](https://user-images.githubusercontent.com/33578163/243127210-026a8b0f-524c-4f76-9c6e-2767dfeb94ef.png) I was wondering if someone could help me to troubleshoot it, please.

Hi,
I think this error may caused by the access permission of specific path on your server. 
I modified the 1.Reads_extract.sh for you, but I am not sure if it can work well on your environment. You can perform a quick test using part of your data.

**Comment by acarmas1** [comment]
Hello,

I've implemented the changes 1.Reads_extract.sh, and I started to get these errors:
![image](https://github.com/weir12/DeepEdit/assets/33578163/4239f147-b2a1-4e60-8466-9241cd597b18)
So I moved all my files to only one directory and ran 1.Reads_extract.sh with only one part of my data. However, it is still taking a while, it's been 2 days that has been running, and it has not finished, but so far no errors, which is good. I'll let you know once it finishes.
